pkg/scalers/prometheus_scaler_test.go

```diff
@@ -370,6 +375,40 @@ func TestPrometheusScalerCustomHeaders(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestPrometheusScalerExecutePromQueryParameters(t *testing.T) {
+	for _, testData := range testPromQueryResult {
+		t.Run(testData.name, func(t *testing.T) {
+			server := httptest.NewServer(http.HandlerFunc(func(writer http.ResponseWriter, _ *http.Request) {
+				writer.WriteHeader(testData.responseStatus)
+
+				if _, err := writer.Write([]byte(testData.bodyStr)); err != nil {
+					t.Fatal(err)
+				}
```

Comment:
Please, validate the query string here to ensure that it's properly set

Reply:
done